core/src/components/datetime/datetime.tsx

```diff
@@ -11,6 +11,9 @@ import type {
   Mode,
   StyleEventDetail,
   TitleSelectedDatesFormatter,
+  DatetimeHighlight,
```

Comment:
~ It might be worth updating these imports to come directly from the `datetime-interface` file located within the same folder vs. the barrel file. Otherwise we will need to manually fix this when syncing `main` with `feature-7.0`. Stencil v3 requires that we import from relative paths instead of the type declaration file that we share with consumers.